src/Publication/Publication.spec.js

```diff
@@ -81,12 +82,8 @@ describe('PublicationService', function() {
   describe('#getModelByType', function() {
     it('should return the model class for the supplied type', function() {
```

Comment:
Same here with the function/arrow inconsistency maybe we can do the opposite and make all the functions use the es5 one instead of arrows